Python 2.7 testing failure due to pandas incompatability

With pandas 0.20.3 and python 2.7.14, testing fails because the `items` attribute does not exist for pandas.Series in this version.

See example output of `python -m pytest`
```
_____________________________________________________________________________________ ERROR at setup of test_run_classifier ______________________________________________________________________________________

    @pytest.fixture
    def db():
        os.remove(DB_PATH)
        db = Database(dialect='sqlite', database=DB_PATH)
        # load cached ModelHub state. This database snapshot has one dataset
        # (pollution_1.csv) and two dataruns, one complete and one with 33/100
        # classifiers finished.
>       db.from_csv(DB_CACHE_PATH)

atm/tests/unit_tests/test_worker.py:64: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
atm/database.py:44: in call
    result = func(db, *args, **kwargs)
atm/database.py:338: in from_csv
    for k, v in list(r.items()):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = id                                                              1
name        ...                                                       7
Name: 0, dtype: object, name = 'items'

    def __getattr__(self, name):
        """After regular attribute access, try looking up the name
            This allows simpler access to columns for interactive use.
            """
    
        # Note: obj.x will always call obj.__getattribute__('x') prior to
        # calling obj.__getattr__('x').
    
        if (name in self._internal_names_set or name in self._metadata or
                name in self._accessors):
>           return object.__getattribute__(self, name)
E           AttributeError: 'Series' object has no attribute 'items'

/usr/local/anaconda3/envs/atm2/lib/python2.7/site-packages/pandas/core/generic.py:3077: AttributeError
```

--- a/atm/database.py
+++ b/atm/database.py
@@ -335,7 +335,7 @@ class Database(object):
 
             for _, r in df.iterrows():
                 # replace NaN and NaT with None
-                for k, v in list(r.items()):
+                for k, v in list(r.iteritems()):
                     if pd.isnull(v):
                         r[k] = None
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 sqlalchemy==1.1.14
 numpy==1.13.1
 boto==2.48.0
-pandas==0.20.3
+pandas==0.22.0
 scikit-learn==0.18.2
 scipy==0.19.1
 sklearn-pandas==1.5.0

--- a/setup.py
+++ b/setup.py
@@ -66,7 +66,7 @@ setup(
         'sqlalchemy>=1.1',
         'numpy>=1.13',
         'boto>=2.48',
-        'pandas>=0.20',
+        'pandas>=0.22',
         'scikit-learn>=0.18',
         'scipy>=0.19',
         'sklearn-pandas>=1.5',

